### ISSUE TITLE
Frontend

### ISSUE THREAD

**Issue by 2xic** [2xic/optimistic-cfd#6]
The project needs a frontend to be user friendly.

I think it's best to use the metamask api to interact with the frontend + smart contract. It's what most people are already familiar with.

In the frontend users has to be (at least) able to do the following
- buy and sell the chip token (stablecoin)
- deposit into a pool
- withdrawal from a pool

If we want to be fancy we can create a more fancy dashboard for the position, but this is not required for the first version.

**Comment by norswap** [comment]
We'll also need at least a way to purchase / sell our stablecoin. Though that could just be an ERC-20 + a pool in an external service for an MVP.

Step 2 would be implementing the pool ourselves (we'll sort of need these components later both frontend or backend).

**Comment by 2xic** [comment]
@norswap Good point! Not sure how I missed that one 😅️  Updated the first post with a new bullet point !

Have you put any thoughts into how the initial coin supply will enter the market ? I was thinking we could mint the chip token in exchange for some other stablecoin (i.e USDC), maybe at a small discount at first since people might not trust the project. 

 If the project takes off, I envision other protocols (i.e Uniswap) might start to offer the chip token.

**Comment by 2xic** [comment]
We should probably also to add support for $C-$CFD AMM in the frontend. To make sure users smoothly can withdraw a position.
